modsecurity/cppbuild.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This file is meant to be included by the parent cppbuild.sh script
+if [[ -z "$PLATFORM" ]]; then
+    pushd ..
+    bash cppbuild.sh "$@" modsecurity
+    popd
+    exit
+fi
+
+
+sudo apt-get install g++ flex bison curl doxygen libyajl-dev libgeoip-dev libtool dh-autoreconf libcurl4-gnutls-dev libxml2 libpcre++-dev libxml2-dev
```

Comment:
The build script should probably not install system dependencies, maybe add a notice on the README mentioning which system dependencies are required?

Reply:
Yep, I agree.

Reply:
Fixed

Reply:
Those should be moved to the scripts for GitHub Actions, if needed.